be/src/runtime/load_channel.cpp

```diff
@@ -64,32 +78,87 @@ void LoadChannel::open(brpc::Controller* cntl, const PTabletWriterOpenRequest& r
         // it will block the bthread, so we put its destructor outside the lock.
         std::shared_ptr<TabletsChannel> channel;
         std::lock_guard l(_lock);
+        if (_schema == nullptr) {
+            _schema.reset(new OlapTableSchemaParam());
+            RETURN_RESPONSE_IF_ERROR(_schema->init(request.schema()), response);
+        }
```

Comment:
I don't quite understand, `LoadChannel::open()` can be called multiple times and each time with a different `request.schema()`, how can the same `_schema` be used for different `TabletsChannel`?

Reply:
all request.schema() are same which come from OlapTableSink.
> tablet_sink.cpp:117
```
request.set_allocated_schema(_parent->_schema->to_protobuf());
```